lib/dropdown.js

```diff
@@ -376,12 +388,15 @@ const positionDropdown = (key, reference) => {
   };
 };
 
+Dropdowns._timeout;
```

Comment:
No need to declare it – you'll attach it further down, at [this line](https://github.com/lookback/meteor-dropdowns/pull/12/files#diff-c9f9fba34f644ab4acff7df7e7d90e73R419) anyway, and JS/Meteor doesn't care if the parameter passed to `Meteor.clearTimeout` is `undefined`.